yhondanilo/chess.rb

```diff
@@ -0,0 +1,41 @@
+class Chess
+
+	attr_accessor :fila, :columna
+
+	def initialize(fila, columna)
+		@fila, @columna=fila, columna
+		intercalar
+	end
+
+	def intercalar
```

Comment:
Assignment Branch Condition size for intercalar is too high. [39.82/15]<br>Tab detected.